src/screens/TokenDetail.js

```diff
@@ -67,6 +70,24 @@ class TokenDetail extends React.Component {
     });
   }
 
+  /**
+   * Fetch NFT info for custom NFT Tokens
+   */
+  fetchNftList() {
```

Comment:
I don't feel this method should be inside the screen. Let's move it somewhere else so we can easily test it. And I also think the best method is `getTokenMetadata(...)` which returns a promise that will be resolved for a json with all metadata available. In this case, we can have another method in this screen called `isNFT(...)` that receives this json and returns a boolean.